packages/path_provider/path_provider_macos/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: macOS implementation of the path_provider plugin
 # 0.0.y+z is compatible with 1.0.0, if you land a breaking change bump
 # the version to 2.0.0.
 # See more details: https://github.com/flutter/flutter/wiki/Package-migration-to-1.0.0
-version: 0.0.4+8
```

Comment:
Why do we need the `-nullsafety` suffix? Nothing has changed in the plugin.

(I'm confused why we need this PR at all at this point, since I wouldn't expect anyone to care if the example of a plugin with no Dart code runs in strict nullsafe mode...)

Reply:
That's a good point. Only raising the minimum SDK version serves no purpose if the package doesn't have any code or dependencies. I'll close the PR